File: lista_0/multiplos/sol.py
def multiplo(a, b):
    maior = a if (a > b) else b 
    menor = a if (a < b) else b 
    print(type(maior/menor), "----", type(maior))
    if(type(maior/menor) == type(maior)): return "Sao multiplos"
    else: return "Nao sao multiplos"
    
# print(multiplo(10,10))

print(10/2)

File: lista_0/multiplos/sol.cpp
#include <iostream>
#include <string>

using namespace std;

string multiplos(int a, int b){
    int maior = (a > b) ? a : b;
    int menor = (a > b) ? b : a;
    if((maior / menor)*menor == maior){
        return "Sao Multiplos\n";
    } 
    return "Nao sao Multiplos\n";
}

int main(){

    int a, b;
    cin >> a >> b;
    cout << (multiplos(a, b));

    return 0;
}